lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -273,7 +278,28 @@ const Reportback = {
         }
       },
     }).done(function(response) {
-      _this.apiUrl = response.meta.pagination.links.next_uri || null;
+      // If response.meta.pagination.links.next, this is from Rogue.
+      if (response.meta.pagination.links.next) {
+        var splitResponseUrl = response.meta.pagination.links.next.split("&");
```

Comment:
Ahh gotcha. We could update that in Rogue by extending `IlluminatePaginatorAdaptor` and changing the little bit that we need, but we'd also need to find a way to pass along the right base URL so the request continues to go through Phoenix as we paginate... could be nice to look into if you want.